mongo/src/main/java/com/redhat/lightblue/mongo/crud/IterateAndUpdate.java

```diff
@@ -160,14 +189,60 @@ public void update(CRUDOperationContext ctx,
                 }
                 docIndex++;
             }
+            // if we have any remaining items to update
+            if (numUpdating > 0) {
+                try {
+                    executeAndLogBulkErrors(bwo);
+                    numFailed += (int) docUpdateErrors.stream().map(e -> e.getIndex()).distinct().count();
```

Comment:
If we want to ignore that case, we can simply just use the size of the error and update attempt lists to set numUpdated and numFailed